lib/modules/apostrophe-images-widgets/public/css/slideshow-item.less

```diff
@@ -23,4 +23,32 @@
       height: auto;
     }
   }
+  &.apos-slideshow--aspect-ratio
+  {
+    .apos-slideshow-item.apos-slideshow-item--svg
+    {
+      background-position: center;
+      img
+      {
+        // Per https://allyjs.io/tutorials/hiding-elements.html#how-to-hide-elements-visually
+        // We need to stop taking up space here, but we'll take it up in inline styles
+        // on the container.
+
```

Comment:
This can be overhauled when the accessibility of the regular images (visibility: hidden) is fixed.